CMakeLists.txt

```diff
@@ -270,15 +263,17 @@ endif(TT_METAL_BUILD_TESTS OR TTNN_BUILD_TESTS)
 # For top level install: cmake --build build --target install  or  make/ninja install -C build
 ############################################################################################################################
 # Install for build artifacts that will upload build/lib
+include(GNUInstallDirs)
```

Comment:
What's this for?

Reply:
This include should provide access to cmake variables such as `CMAKE_INSTALL_PREFIX`

Reply:
https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html